lib/membrane/video_compositor/compound/layout.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Membrane.VideoCompositor.Compound.Layout do
+  @moduledoc """
+  Behaviour representing layous - modification combing multiple objects into one.
+
+  Render callback is a placeholder.
+  """
+
+  @type definition_t :: struct() | module()
```

Comment:
Are you sure we want to stick to the convention of types having `_t` suffix? Recently we have decided to get rid of all the `_t` suffixes in Membrane Core as we have realised nobody is following such a convention - but I don't know if there is any "Membrane-wide" agreement on that :D

Reply:
I like type names without `_t` - let's talk about Membrane-wide convention on the next brainstorming